api/src/main/proto/spine/protodata/source.proto

```diff
@@ -42,7 +42,7 @@ import "spine/protodata/ast.proto";
 // Includes all the message types declared in the file, along with their fields.
 //
 message ProtobufSourceFile {
-    option (entity).kind = PROJECTION;
+    option (entity).kind = VIEW;
 
     FilePath file_path = 1;
```

Comment:
We probably need to document the id fields here saying that its value should be equal to one from the `file.path` property.

Reply:
Added [an issue](https://github.com/SpineEventEngine/validation/issues/83) for the `same_as` option.